src/world/terrain/generation/land_generator.hpp

```diff
@@ -258,41 +264,20 @@ class LandGenerator {
     };
 };
 
-enum class AddDirections : uint8_t {
-    None = 0,
-    To = 1,
-    Add = 2,
-};
-
-struct AddMethod {
-    // enum class about add n vs to n
-
-    Dim start;
-    Dim stop;
-    Dim data;
-
-    AddDirections add_directions;
-
-    bool
-    operator<(const AddMethod& other) const {
-        return start < other.start;
-    }
-};
-
 class AddToTop {
     const MaterialGroup elements_above_;
 
     const MaterialGroup elements_can_overwrite_;
     const MaterialId stamp_material_id_; // MaterialId that will be set
     const ColorId stamp_color_id_;       // ColorId that will be set
 
-    std::set<AddMethod> data_;
+    std::set<layer_effect_data_t> data_;
```

Comment:
unordered

Reply:
1) wrong pr
2) I used the ordering somewhere. I think I will again to determine the color a tile should be.